### PR TITLE
 Pull Request 1: Header Feature

### PR DESCRIPTION
**Pull request by ZikaZaki**
### In this pull-request I:

- [x] Configured the project structure
- [x] Configured the **Linters** using github-workflows:
     - [x] Lighthouse
     - [x] Webhint
     - [x] Stylelint
- [x] Created a descriptive README
- [x] Documented the repo based on **best practices**
- [x] Followed **best practices for HTML & CSS** 
- [x] Parsed & used **Template 1** Figma design file
- [x] Created the toolbar (or header)
- [x] Created the headline section (right after the header)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <i id="menu" class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <div class="text-holder">
+                <p class="primary-txt">
+                    I'm Zakariya
+                    Glad to see <br>you!
+                </p>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Please make a change! The main title (e.g., "I'm Marianne ...") should be a `h1` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                </svg>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon should has a link (`a` tag). Please make changes!

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @ZikaZaki , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good mobile design ✔️ 
- Good Implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/index.css">
+    <title>ZikaZaki-Folio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="#" class="logo">My Logo</a>
+            <i id="menu" class="fa-solid fa-bars"></i>
```
Comment (Ismailco) [inline index.html:16]: - [x] [OPTIONAL] The hamburger menu should include a `nav` tag, kindly implement this

**Review by Ismailco — APPROVED** [review]
### Hi @ZikaZaki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (ZikaZaki) [reply index.html:16]: The **nav** tag is already included. Thanks for the review.

**Pull request merged**